src/index.html

```diff
@@ -1,352 +1,459 @@
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>GC Notify - System Status</title>
+    <meta name="description" content="GC Notify system status" />
+    <meta name="keywords" content="" />
+    <meta name="author" content="" />
 
-<head>
-  <meta charset="UTF-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-  <title>GC Notify - System Status</title>
-  <meta name="description" content="GC Notify system status" />
-  <meta name="keywords" content="" />
-  <meta name="author" content="" />
+    <!--Icons-->
+    <link
```

Comment:
Phosphor doesnt seem to have an animated spinning class, so just a note that we'll need to add one.

Reply:
We could probably make our own spinner class while using one of the spinner icons from phosphor.